client/site-profiler/components/hosting-information/index.tsx

```diff
@@ -11,16 +14,75 @@ interface Props {
 export default function HostingInformation( props: Props ) {
 	const { dns = [], hostingProvider } = props;
 	const aRecordIps = dns.filter( ( x ) => x.type === 'A' && x.ip );
+	const [ popoverVisible, setPopoverVisible ] = useState( false );
+	const hostingProviderNameRef: React.RefObject< HTMLDivElement > = useRef( null );
+
+	const isPopularCdn = !! hostingProvider?.is_cdn;
+	const getNonAutomatticHostingElement = () => {
+		return isPopularCdn ? (
+			<Button
+				borderless={ true }
+				className="action-buttons__borderless hosting-provider-name"
+				href="#"
+			>
+				{ hostingProvider?.name }
+			</Button>
+		) : (
+			hostingProvider?.name
+		);
+	};
 
 	return (
 		<div className="hosting-information">
 			<h3>Hosting information</h3>
```

Comment:
If you have a chance, please wrap this string with a translate.
Thanks in advance! 🙏